### PR TITLE
Add state_color support to entity card

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Currently, the frontend is a bit inconsistent since we cannot enable `state_color` for an entity card, so the look is different from e.g. glance or entities cards.

This PR now adds the support to both the card and the editor.

grafik

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/19229

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by spacegaier** [comment]
I will add documentation PR once we agree that this feature is worth adding.

File: src/panels/lovelace/cards/hui-entity-card.ts [hunk 1]
```diff
@@ -238,6 +250,8 @@ export class HuiEntityCard extends LitElement implements LovelaceCard {
         font-size: 18px;
         color: var(--secondary-text-color);
       }
+
+      ${iconColorCSS}
```
Comment (bramkragten) [inline src/panels/lovelace/cards/hui-entity-card.ts:254]: you should return an array, with `iconColorCSS` and the `css` above

**Comment by bramkragten** [comment]
I think aligning makes sense...

File: src/panels/lovelace/cards/hui-entity-card.ts (hunk 1 above)
Comment (spacegaier) [reply src/panels/lovelace/cards/hui-entity-card.ts]: Done. Found two other places where I did the same for consistency.

**Review by bramkragten — APPROVED** [review]

**Pull request merged**